src/agents/coordinator_agent.py

```diff
@@ -80,21 +83,39 @@ class CheckParkingOffers(CyclicBehaviour):
         """Behaviour for checking parking offers"""
 
         def _check_if_request_is_within_region(self, request_x, request_y):
-            return self.agent._x_min <= request_x < self.agent._x_max and self.agent._y_min <= request_y < self.agent._y_max
+            return (
+                self.agent._x_min <= request_x < self.agent._x_max
+                and self.agent._y_min <= request_y < self.agent._y_max
+            )
 
         async def run(self):
-            msg = await self.receive(timeout=MESSAGE_TIMEOUT)
+            # msg = await self.receive(timeout=MESSAGE_TIMEOUT)
```

Comment:
Is it for tests purposes only?

Reply:
yes, I was trying to somehow see how the messages are being sent between agents